massgen/orchestrator_collaborators/runtime_input_delivery.py

```
@@ -0,0 +1,460 @@
+            if target_agents:
+                candidate_agent_ids = [aid for aid in target_agents if isinstance(aid, str) and aid in orch.agents]
+            else:
+                candidate_agent_ids = list(orch.agents.keys())
+
+            for candidate_agent_id in candidate_agent_ids:
+                loop.create_task(
+                    orch._maybe_interrupt_background_wait_for_agent(
+                        candidate_agent_id,
+                        trigger="queued_human_input",
+                    ),
+                )
```

Ruff (0.15.14)

[warning] Store a reference to the return value of `loop.create_task`

(RUF006)